templates/app/app-secrets.yaml

```diff
@@ -1,18 +1,20 @@
 apiVersion: v1
 kind: Template
 metadata:
-  name: manageiq-secrets
+  name: app-secrets
```

Comment:
Should these be `${APP_NAME}-secrets` as well?

Reply:
Nope, explained that in the first commit message actually.

> It's hard to productize based on a configurable app name when
we're referencing the object name in the deploy script.
This avoids the problem by not using the app name in the secret name
at all.